azext_edge/edge/_help.py

```diff
@@ -650,6 +650,67 @@ def load_iotops_help():
         short-summary: Dataflow endpoint management.
     """
 
+    helps[
+        "iot ops dataflow endpoint apply"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow endpoint resource.
+        long-summary: |
+          An example of the config file format is as follows:
+
+          {
+            "properties": {
+              "endpointType": "Kafka",
+              "kafkaSettings": {
+                "authentication": {
+                  "method": "SystemAssignedManagedIdentity",
+                  "systemAssignedManagedIdentitySettings": {
+                    "audience": "aio-internal"
+                  }
+                },
+                "batching": {
+                  "latencyMs": 5,
+                  "maxBytes": 1000000,
+                  "maxMessages": 100000,
+                  "mode": "Enabled"
+                },
+                "cloudEventAttributes": "Propagate",
+                "compression": "None",
+                "copyMqttProperties": "Disabled",
+                "host": "*.servicebus.windows.net:9093",
+                "kafkaAcks": "All",
+                "partitionStrategy": "Default",
+                "tls": {
+                  "mode": "Enabled"
+                }
+              },
+            }
+          }
+
+          When used with apply the above content will create or replace a target kafka dataflow endpoint
```

Comment:
we may want to warn users if the dataflow exists and will be overwritten (with a confirm prompt)

Reply:
This is an apply desired state command, it is intended to do what is necessary to make the resource looks like this. Draw a parallel to `kubectl apply` which does not error or prompt if the intended resource already exists.

Reply:
mostly wanted to make sure we don't let users accidently overwrite but tbh wasn't sure if that is too much hand holding

Reply:
With kubectl it was a conscious decision, because `kubectl create` will raise an error if the resource exists.